### PR TITLE
Remove ternary and attach headline variant to page only

### PR DESCRIPTION
**Pull request by missamynicholson**
@tom-parker

### CODE REVIEW

**Comment by tom-parker** [comment]
👍 

Unfortunately I think the logic in o-tracking that moves 'headline-test' and the relevant value into the context is going to change so this is likely to break in the future

File: components/n-ui/tracking/ft/index.js [hunk 1]
```diff
@@ -104,9 +106,17 @@ const oTrackingWrapper = {
...
-			//headline testing
-			const headlineTestingVariant = document.querySelector('[data-trackable-headline-variant]') ? document.querySelector('[data-trackable-headline-variant]').getAttribute('data-trackable-headline-variant') : null;
-			context['headline-variant'] = headlineTestingVariant;
+			//headline testing, add variant to the page view event as long as there is only one article under test
+			if (location.pathname === '/') {
+				const alternativeHeadlines = document.querySelectorAll('[data-trackable-headline-variant]');
+				let hrefs = [];
+				alternativeHeadlines.forEach(headline => hrefs.push(headline.getAttribute('href')));
+
+				const isOnlyOneArticle = hrefs.length ? !!hrefs.reduce((a, b) => a === b ? a : NaN) : false;
```
Comment (tom-parker) [inline components/n-ui/tracking/ft/index.js]: Maybe I've missed something but why `NaN` here?

Comment (missamynicholson) [reply components/n-ui/tracking/ft/index.js]: Good point, could have just been false.
Have updated it to slice the NodeList as you suggested, and changed the href equality check to use Array.every() as it's a bit neater.

**Comment by tom-parker** [comment]
👍

**Pull request merged**